docs/source/index.rst

```diff
@@ -65,9 +65,9 @@ Usage
    :hidden:
 
    usage/how_to_run
+   usage/examples
```

Comment:
Moved forward as in WarpX: showing examples before API details is more user-friendly :)